packages/transactions/src/TokenBurnV1.ts

```diff
@@ -62,14 +62,15 @@ export default class TokenBurnV1 extends Transaction {
   private toProto(forSigning: boolean = false): proto.helium.blockchain_txn_token_burn_v1 {
     const TokenBurnTxn = proto.helium.blockchain_txn_token_burn_v1
     const memoBuffer = Buffer.from(this.memo, 'base64')
+    const memoLong = JSLong.fromBytes(Array.from(memoBuffer), true, true)
     return TokenBurnTxn.create({
       payer: toUint8Array(this.payer.bin),
       payee: toUint8Array(this.payee.bin),
       amount: this.amount,
       nonce: this.nonce,
       signature: this.signature && !forSigning ? toUint8Array(this.signature) : null,
       fee: this.fee,
-      memo: JSLong.fromBytes(Array.from(memoBuffer), true, true),
+      memo: !memoLong.isZero() ? memoLong : undefined,
```

Comment:
Could `memoLong` be undefined? If so maybe it just needs an extra check? 

`memo: memoLong && !memoLong.isZero() ? memoLong : undefined,`

Reply:
It looks like it should always return Long, not sure if the check is needed https://github.com/dcodeIO/long.js/blob/master/README.md